package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "debug": "^2.1.3",
     "glob": "^5.0.3",
+    "parent-require": "^1.0.0",
```

Comment:
little mixed on making `parent-require` as a dependency coz __its used/will be used__ only while you are using `react-engine` via `npm link` (which happens to be only in some development mode of react-engine). but i''ll take this given the fact we use `parent-require` explicitly in `util.js`